modules/persistent-storage-cinder-creating-pv.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * storage/persistent_storage/persistent-storage-cinder.adoc
+
+[id="persistent-storage-cinder-creating-pv_{context}"]
+= Creating the Persistent Volume
+
+You must define your persistent volume (PV) in an object definition before creating
+it in {product-title}:
+
+.Procedure
+
+. Save your object definition to a file.
++
+.azure-advanced-disk-storageclass.yaml
+[source,yaml]
+----
+apiVersion: "v1"
+kind: "PersistentVolume"
+metadata:
+  name: "pv0001" <1>
+spec:
+  capacity:
+    storage: "5Gi" <2>
+  accessModes:
+    - "ReadWriteOnce"
+  cinder: <3>
+    fsType: "ext3" <4>
+    volumeID: "f37a03aa-6212-4c62-a805-9ce139fab180" <5>
+----
+<1> The name of the volume that is used by persistent volume claims or pods.
+<2> The amount of storage allocated to this volume.
+<3> The volume type.
+<4> Filesystem-type to mount.
```

Comment:
Filesystem -> file system.

I'd rewrite this to say:

> The file system that is created when the volume is mounted for the first time.

Reply:
It's in the IBM Style Guide to use it as two words, thus the suggestion here. I've been guilty of writing it as one, though. :)